dips/dip-10.md

```diff
@@ -101,130 +110,87 @@ struct DiemIdDomainEvent {
-   "amount": "<amount>",
-   "currency": "<currency>"
-}
+The format of the success response is:
 ```
-
-| Field 	   | Type 	     | Required? 	| Description 	           |
-|-------	   |------	     |-----------	|-------------	           |
-|sender        |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object)  | Y            | Information about the sender |
-|receiver_id   |str          | Y            | Diem ID of the receiver |
-|amount        |integer      | Y            | Amount of coins to transfer |
-|currency      |str          | Y            | Name of the coin |
-
-**Response:**
-```json
 {
-   "receiver_address": "<Receiver VASP on-chain account address>",
-   "reference_id": "<Payment_Reference_ID>",
-   "receiver": "<User Object>"
+   "_ObjectType": "CommandResponseObject",
+    "status": "success",
+    "result": {
+	    "_ObjectType": "ReferenceIDCommandResponse",
+	    "recipient_address": "c5ab123458df0003415689adbb47326d",
+    },
+    "cid": "12ce83f6-6d18-0d6e-08b6-c00fdbbf085a",
 }
 ```
 
-| Field 	      | Type       | Required? 	| Description 	           |
-|-------	      |------      |-----------	|-------------	           |
-|receiver_address |str         | Y            | On-chain account address for receiving the payment. This address must be encoded as a [Bech32 account identifier](https://github.com/diem/dip/blob/master/dips/dip-5.md#account-identifiers) **without** subaddress |
-|reference_id     |str         | Y            | Reference ID to be included into payment metadata |
-|receiver         |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object) | Y            | Information about the receiver |
-
-* Pay endpoint is used to exchange details of the payment such as information about Sender and Receiver off the chain, and negotiate `Payment_Reference_ID`
-
-* Once the sender learns Payment_Reference_ID, it will submit an on-chain transaction including the `Payment_Reference_ID` as a metadata. Receiver will use the `Payment_Reference_ID` in the transaction to link the on-chain transaction to the payment.
+**CommandResponseObject:**
 
-* Sender will submit a transaction to the `receiver_address` on-chain account in the response. This account address can be either Parent VASP, or any of Child VASP accounts of the receiver VASP. This is needed to allow the receiver VASP to manage load between different on-chain accounts.
+| Field 	    | Type 	     | Required? 	| Description 	           |
+|-------	    |------	     |-----------	|-------------	           |
+| _ObjectType   | str        | Y | Fixed value: `CommandResponseObject`|
+| status       | str | Y | Either `success` or `failure`. |
+| result       | Result object | Y | The Result obejct of response. |
+| cid           | str         | Y            | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included. |
```

Comment:
why do we need `cid` in response? the sender should have request's cid when they get the response

Reply:
just echoing dip1